### ISSUE TITLE
Calling out for Data science people!💻📈

### ISSUE THREAD

**Issue by pratik-choudhari** [pratik-choudhari/AlgoCode#162]
# Current situation
As of now this repository has been accepting contributions for solving coding problems. Which is going pretty good.

# So now what?
Now we are accepting contributions for some beautiful and insightful EDAs on dataset of your choice. Yes! a dataset of your choice.
The rules for EDA contribution:
* Jupyter notebook must go inside `EDA/<your-dataset-name>`
* It is possible to 2 people make PRs simultaneously, to avoid such clutter First check if the dataset directory exists, else post a comment with dataset name on [this issue](https://github.com/pratik-choudhari/AlgoCode/issues/161)
* Mandatory to add a README or append to existing README 
* If a PR is made without abiding by these rules, PR will be marked `spam`.

__You may leave any queries below__

**Comment by Mrsterius** [comment]
Hi Pratik,
I would like to add the EDA for Boston Housing dataset. Can I work on this?

**Comment by pratik-choudhari** [comment]
@Mrsterius Yes

**Comment by Mrsterius** [comment]
Hey Pratik,
I have created EDA for the AMES Advanced house prices (regression) dataset since column names were not mentioned for the Boston one. What all do you want me to add in the README?

**Comment by Mrsterius** [comment]
Submitted. Please let me know if there any changes to be made.

**Comment by PrattJena** [comment]
Hey Pratik
I would like to add an EDA of dogs vs cats. Can i work on it?

**Comment by pratik-choudhari** [comment]
@PrattJena What in cats and dogs? The image dataset right?

**Comment by PrattJena** [comment]
@pratik-choudhari Yes the image classification dataset

**Comment by pratik-choudhari** [comment]
Could you elaborate on what will be in the EDA?

**Comment by PrattJena** [comment]
Actually now that I think about it. Its much better suited for projects folder as the Neural network determines whether the image is of cat or a dog

**Comment by pratik-choudhari** [comment]
Yep so that doesn't count as EDA.

**Comment by rajpratyush** [comment]
EDA for Fashion MNIST dataset and CIFAR 10 data set

**Comment by pratik-choudhari** [comment]
@rajpratyush What will be the contents in your EDA?

**Comment by rajpratyush** [comment]
I have done several assignments on these two datasets while learning through an MOOC on Coursera Platform. I would like to share those.

**Comment by pratik-choudhari** [comment]
@rajpratyush EDA on a image dataset, won't it be vague? On a dataset like CIFAR 10 the maximum one could get from EDA is the number of images in every category, correct me if there is any addition to this.

**Comment by rajpratyush** [comment]
I agree then how about an EDA ON a data set of toxic words  i remeber there was a kaggle competition regarding this  haad a successful contribution in this

**Comment by pratik-choudhari** [comment]
@rajpratyush Yes that will suffice.

**Comment by rajpratyush** [comment]
@pratik-choudhari btw you have assigned me this issue yet https://github.com/pratik-choudhari/AlgoCode/issues/94#issue-714136213

**Comment by drashtipatel2503** [comment]
Hii, can I go for weather analysis?

**Comment by pratik-choudhari** [comment]
@drashtipatel2503 Sure! Put a link and name on data set on issue #161

**Comment by sanskritip** [comment]
Hey Pratik! Can I add EDA for StackOverflow Developer 2019 Dataset? I will be adding the CSV files, Data Cleaning Jupyter Notebook and additionally the Visualisation Notebook.

**Comment by pratik-choudhari** [comment]
@sanskritip Sure, don't include the CSV files it will increase the repo size instead put the link of dataset.

**Comment by sanskritip** [comment]
> 
> 
> @sanskritip Sure, don't include the CSV files it will increase the repo size instead put the link of dataset.

Awesome Sounds great!
